tools/text_processing/text_processing/unsorted_uniq.xml

```diff
@@ -52,6 +63,24 @@
             </conditional>
             <output name="outfile" file="unique_results1.bed"/>
         </test>
+        <test>
+            <param name="infile" value="1.bed"/>
+            <param name="is_numeric" value="True"/>
+            <param name="ignore_case" value="True"/>
+            <conditional name="adv_opts">
+                <param name="adv_opts_selector" value="advanced"/>
+                <param name="column_start" value="2"/>
+                <param name="column_end" value="3"/>
+                <param name="header" value="1"/>
+            </conditional>
+            <output name="outfile" file="unique_results2.bed"/>
+        </test>
+        <test>
+            <param name="infile" value="1_dup.bed"/>
```

Comment:
This test could be made more conclusive if the test input file didn't start with the smallest value. It would then demonstrate that without the advanced section the first line gets included in the sort.